tools/depends/target/ffmpeg/Makefile

```diff
@@ -20,15 +20,13 @@ ffmpg_config += --enable-encoder=ac3 --enable-encoder=aac
 ffmpg_config += --enable-encoder=wmav2 --enable-protocol=http
 ffmpg_config += --enable-gnutls
 ffmpg_config += --enable-encoder=png --enable-encoder=mjpeg
+ffmpg_config += $(FFMPEG_CONFIGURE_OPTIONS)
 
 ifeq ($(CROSS_COMPILING), yes)
   ffmpg_config += --arch=$(CPU) --enable-cross-compile
 endif
 ifeq ($(OS), linux)
-  ffmpg_config += --target-os=$(OS) --cpu=$(CPU)
-  ifneq ($(TARGET_PLATFORM),raspberry-pi)
-    ffmpg_config += --enable-vdpau --enable-vaapi
-  endif
+  ffmpg_config += --target-os=$(OS)
```

Comment:
we should probably still pass a CPU if ffmpeg-flags was not given, maybe even if

Reply:
this would mean a clear cpu / arch rework (again)
for arm builds cpu=arm which is wrong and leads to errors in ffmpeg builds

Reply:
why would cpu be arm? As is ffmpeg won't know which cpu to choose.
Maybe we need another configure switch for cpu that can override the defaults.

Reply:
because we need --with-arch=arm and we use cpu here to fill --with-arch.
This was my initial arch / cpu PR some time ago (splitting arch and cpu) because we cannot handle both using one variable.

Reply:
Side note: ffmpeg configure does many checks for CPU already, IMO there is no difference if you set --with-cpu as configure option or pass -mcpu as C(XX) flag

Reply:
you just add -cpu to x86 in configure if nothing was set by the user